dev-ruby/vagrant_cloud/vagrant_cloud-2.0.1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+USE_RUBY="ruby23 ruby24 ruby25"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_GEMSPEC="${PN}.gemspec"
+RUBY_FAKEGEM_RECIPE_TEST="rspec3"
+RUBY_FAKEGEM_TASK_DOC=""
```

Comment:
We can probably remove this.

Reply:
nope, removing it adds `doc` useflag, but I haven't tested what happens If I actually enable it =D
probably nothing good.

Reply:
```
rake aborted!
LoadError: cannot load such file -- bundler/gem_tasks
/tmp/portage/dev-ruby/vagrant_cloud-2.0.1/work/all/vagrant_cloud-2.0.1/Rakefile:3:in `<top (required)>'
(See full trace by running task with --trace)
 * ERROR: dev-ruby/vagrant_cloud-2.0.1::gentoo failed (compile phase):
 *   failed to (re)build documentation
```

yep, looks like there is no docs there.